File: uniapp_flutter/lib/models/url_end_points.dart
class Urlendpoints {
  String rootconfig;
  String login;
  String projecttype;
  String projectlist;
  String syncdata;
  String logout;

  Urlendpoints(
      {this.rootconfig,
        this.login,
        this.projecttype,
        this.projectlist,
        this.syncdata,
        this.logout});

  Urlendpoints.fromJson(Map<String, dynamic> json) {
    rootconfig = json['rootconfig'];
    login = json['login'];
    projecttype = json['projecttype'];
    projectlist = json['projectlist'];
    syncdata = json['syncdata'];
    logout = json['logout'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['rootconfig'] = this.rootconfig;
    data['login'] = this.login;
    data['projecttype'] = this.projecttype;
    data['projectlist'] = this.projectlist;
    data['syncdata'] = this.syncdata;
    data['logout'] = this.logout;
    return data;
  }
}

File: app/app/src/main/java/com/vassar/unifiedapp/model/URLEndpoints.java
package com.vassar.unifiedapp.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
@JsonIgnoreProperties(ignoreUnknown = true)
class URLEndpoints {
    @JsonProperty("rootconfig")
    public String mRootConfigEndpoint;
    @JsonProperty("login")
    public String mLoginEndpoint;
    @JsonProperty("projecttype")
    public String mProjectTypeEndpoint;
    @JsonProperty("projectlist")
    public String mProjectListEndpoint;
    @JsonProperty("syncdata")
    public String mSyncDataEndpoint;
    @JsonProperty("logout")
    public String mLogoutEndpoint;

    public URLEndpoints() {
    }

    @Override
    public String toString() {
        return "URLEndpoints{" +
                "mRootConfigEndpoint='" + mRootConfigEndpoint + '\'' +
                ", mLoginEndpoint='" + mLoginEndpoint + '\'' +
                ", mProjectTypeEndpoint='" + mProjectTypeEndpoint + '\'' +
                ", mProjectListEndpoint='" + mProjectListEndpoint + '\'' +
                ", mSyncDataEndpoint='" + mSyncDataEndpoint + '\'' +
                ", mLogoutEndpoint='" + mLogoutEndpoint + '\'' +
                '}';
    }
}
